README.md

```diff
@@ -14,6 +14,34 @@ Step-by-step instructions on getting Assemble running with RHSSO authentication
 1. Openshift 4.9+
 1. Helm 3+
 
+### Vault
+Install the vault operator which will be used for the postresql password and other secrets related to Backstage. In addition applications will make use of the vault for any sensitive data.
+
+#### Install Cert Manager Operator
+The vault server uses a certificate generated by the cert-manager operator. If the operator is not already installed install the opertor.
+
+```
+# From the charts/operator directory.
+helm upgrade --install cert-manager . -f ./values-certmanager-operator.yaml -n openshift-operators
+```
+
+#### Install Vault Config Operator
+The vault config operator is used to manage policies and secrets in the vault instance.
+
+```
+# From the charts/operator directory.
+helm upgrade --install vault-config-operator . -f ./values-vault-config-operator.yaml -n vault-config-operator --create-namespace
+```
+
+#### Install the Vault operator
+
```

Comment:
I had to do a `helm dep update` in order to get this next step to run correctly.  You may want to add that as a step here